tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -1289,6 +1291,14 @@ TEST_F(ValidationTest, I8Type) {
     /*bRegex*/true);
 }
 
+TEST_F(ValidationTest, EmptyStructInBuffer) {
+  TestCheck(L"..\\CodeGenHLSL\\staticGlobals.hlsl");
```

Comment:
This looks like a copy-paste error. Should probably point to the EmptyStructInBuffer test.